cmd/juju/action/run.go

```diff
@@ -97,12 +99,17 @@ func (c *runCommand) SetFlags(f *gnuflag.FlagSet) {
 }
 
 func (c *runCommand) Info() *cmd.Info {
-	return jujucmd.Info(&cmd.Info{
+	info := jujucmd.Info(&cmd.Info{
 		Name:    "run-action",
 		Args:    "<unit> [<unit> ...] <action name> [key.key.key...=value]",
 		Purpose: "Queue an action for execution.",
 		Doc:     runDoc,
 	})
+	if featureflag.Enabled(feature.JujuV3) {
+		info.Name = "run"
+		info.Doc = strings.Replace(info.Doc, "run-action", "run", -1)
```

Comment:
I think it would be good to append a note to the doc like "NOTE: pre-3.0 juju run functionality is provided by juju exec"

Reply:
I don't thin we need to do this because this v3 behaviour is strictly opt-in - the user has to enable a feature flag to see it. They are therefore already aware of what they are doing in that sense and what the command names are etc.